src/main/resources/plugin.yml

```diff
@@ -1,14 +1,106 @@
 name: Wild
-version: '2.10'
+version: 2.11

```

Comment:
Don't bump pls.

You'll want to leave it as a string regardless, else it'll cut off trailing zeroes.

Reply:
k, I've pushed this huge modif ;)

BTW, never had this issue... a cleaner way not to cut off 0 would be to use a maven variable in the plugin.yml, and put the version number in pom.yml.

Reply:
It's because the yml parser in CB will interpret it as a double, and thus cut off trailing 0s after the decimal point.